python/ray/data/_internal/progress_bar.py

```diff
@@ -44,15 +44,22 @@ class ProgressBar:
     because no tasks have finished yet), doesn't display the full
     progress bar. Still displays basic progress stats from tqdm."""
 
+    # If the name/description of the progress bar exceeds this length,
+    # it will be truncated.
+    MAX_NAME_LENGTH = 100
+
     def __init__(
         self,
         name: str,
         total: Optional[int],
         unit: str,
         position: int = 0,
         enabled: Optional[bool] = None,
+        display_full_name: bool = False,
     ):
-        self._desc = name
+        # If True, disables name trunctating.
+        self._display_full_name = display_full_name
+        self._desc = self._truncate_name(name)
```

Comment:
How would this be set to True by the user? Is the expectation that they would not want that or should we have something in data context or execution options to allow for this configuration?

Reply:
ah yeah good point, i will need to expose it from DataContext